test/e2e/specs/editor/various/preview.spec.js

```diff
@@ -184,6 +184,20 @@ test.describe( 'Preview', () => {
 		// FIXME: The confirmation dialog is not named yet.
 		await page.click( 'role=dialog >> role=button[name="OK"i]' );
 
+		// Wait for the status change.
+		await expect(
+			page.locator( 'role=button[name="Publish"i]' )
+		).toBeVisible();
+
+		const previewToggle = page.locator(
+			'role=button[name="View"i][expanded=true]'
```

Comment:
Nit and my own preference:
```suggestion
			'role=button[name="View"i][expanded]'
```